x-pack/plugins/drilldowns/public/components/drilldown_hello_bar/drilldown_hello_bar.scss

```diff
@@ -0,0 +1,7 @@
+@include internetExplorerOnly {
+  .drdHelloBar__content .drdHelloBar__contentWrapIE {
+    // flex-basis: auto doesn't wrap properly,
+    // so have to use static value that works for most cases
+    flex-basis: 480px;
```

Comment:
flex-basis: auto; didn't work :( 
flex-container overlapped the flyout causing endless horizontal scroll, as text inside is not wrapping.
@elastic/kibana-design if there is a magic tip, I'd appreciate it 🙏

Reply:
<img width="1079" alt="Screenshot 2020-05-07 at 15 12 19" src="https://user-images.githubusercontent.com/7784120/81298541-32701100-9075-11ea-9de9-78c8edeb92cb.png">
this is what happens with `auto

Reply:
I checked at the flyout banner example from our EUI docs in IE, and it shows the layout working correctly. I compared to your instance of EuiCallOut, and noticed that you're putting all the content in the `title` of the EuiCallout instead passing it as the `children`. If you pass flex group and contents as the `children`, you shouldn't have any IE issues.

Reply:
Thanks @cchaos, it helped!

<img width="891" alt="Screenshot 2020-05-08 at 12 46 20" src="https://user-images.githubusercontent.com/7784120/81399281-09aa5300-912b-11ea-951e-3d1b79e08466.png">

Reply:
@cchaos, mind hitting ✅ for Kibana-design?